pom.xml

```diff
@@ -41,6 +41,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <camel.version>3.0.0-SNAPSHOT</camel.version>
+        <hapi.version>3.7.0</hapi.version>
```

Comment:
Can we not use the hapi version defined in the camel release, to avoid having endless of new version management here too

Reply:
Right now the FHIR R4 data structures aren't part of the Camel release. My plan was to add it in Camel once this has been merged.  Once updated this will be removed. Thanks !

Reply:
Hi @lburgazzoli, yes I have to redefine the version in camel-quarkus as properties cannot be imported from another POM in Maven as far as I know ? Thanks !

Reply:
Yes temporarily, which is confusing. I thought I'd do everything in one go, let me know if that is okay

Reply:
I have no objection as long as it is clearly documented and we catch up on camel asap